website/addons/base/views.py

```diff
@@ -577,7 +577,7 @@ def addon_deleted_file(auth, node, error_type='BLAME_PROVIDER', **kwargs):
         'provider': file_node.provider,
         'materialized_path': file_node.materialized_path or file_path,
         'private': getattr(node.get_addon(file_node.provider), 'is_private', False),
-        'file_tags': [tag._id for tag in file_node.tags],
+        'file_tags': file_node.tags.all().values_list('id', flat=True),
```

Comment:
I can't remember if files could have system tags prior, but system tags and tags are all in the tags bucket now, so they can now. To get only non system tags `file_node.tags.filter(system=False).values_list('id', flat=True)`